build.psake.ps1

```diff
@@ -243,7 +243,11 @@ Task Test -depends Build {
     Import-Module Pester
     try {
         Microsoft.PowerShell.Management\Push-Location -LiteralPath $TestRootDir
-        Invoke-Pester
+        $TestResult = Invoke-Pester -PassThru -Verbose:$VerbosePreference
```

Comment:
Pester still writes out the test output even when -PassThru is being used.  I just tried it when messing with the AppVeyor stuff, works fine.